http/src/http.mli

```diff
@@ -410,6 +410,14 @@ module Request : sig
 
       See https://www.rfc-editor.org/rfc/rfc7230#section-3.3.2 *)
 
+  val supports_chunked_trailers : t -> bool
+  (** [supports_chunked_trailers t] is [true] if [t] contains HTTP header "TE:
+      trailers". Otherwise it is [false]. *)
+
+  val add_te_trailers_header : t -> t
```

Comment:
`_header` is redundant here
```suggestion
  val add_te_trailers : t -> t
```
You Are clearly modifying the header from the signature

Reply:
Done.